cypress/integration/Train/ScenarioAndTags.spec.js

```diff
@@ -0,0 +1,149 @@
+import * as models from '../../support/Models'
+import * as modelPage from '../../support/components/ModelPage'
+import * as editDialogModal from '../../support/components/EditDialogModal'
+import * as train from '../../support/Train'
+import * as helpers from '../../support/Helpers'
+
+describe('Training Scenario and Tags', () => {
```

Comment:
The title makes it sound like your going to learning/training the scenario.  It's just manually typed and edited by a human, no training involved. I would remove the word or make it Train Dialog since that's what we call it.

Reply:
done